declarations_site/catalog/static/js/decl-compare.js

```diff
@@ -0,0 +1,219 @@
+//read decl.ID from localstorage and populate compare list
+function getCompareList() {
+    var c = 0;
+
+    $.each(localStorage, function(key, value){
+        if (key.indexOf('declarationID-') >= 0) {
+            c++;
+            $('.decl-compare-toggle').removeClass('hidden');
+
+            var declarationID = key.substring(14);
```

Comment:
I do understand that, but:

1. Make it constant.
2. Explain it. You may use comments or you may calculate it, like this: my_magic_number = "my_string_prefix".length;